angular-workspace/projects/ni/nimble-angular/rich-text-editor/public-api.ts

```diff
@@ -0,0 +1,2 @@
+export * from './nimble-rich-text-editor.directive';
```

Comment:
Could you also export the page object in its own `testing` entry point [like the table does](https://github.com/ni/nimble/blob/main/angular-workspace/projects/ni/nimble-angular/table/testing/public-api.ts)? That will allow Angular clients to use it to write their unit tests while still only depending on `nimble-angular`, not `nimble-components`. 

Looks like we forgot to do this for the rich-text-viewer so let's add a testing entry point for it too.

Reply:
Added for both the components.